climada/hazard/tc_tracks.py

```diff
@@ -865,9 +865,9 @@ def equal_timestep(self, time_step_h=1, land_params=False):
                                                       land_geom))
             self.data = new_data
 
-    def calc_random_walk(self, **kwargs):
+    def calc_random_walk(self, nb_synth_tracks=9, **kwargs):
```

Comment:
This is the API function that has been available for several CLIMADA versions. I would rename it like this:
```
def calc_random_walk(self, *args, **kwargs):
    LOGGER.warn("The use of TCTracks.calc_random_walk is deprecated."
                "Use TCTracks.calc_perturbed_trajectories instead.")
    return self.calc_perturbed_trajectories(self, *args, **kwargs)

def calc_perturbed_trajectories(self, nb_synth_tracks=9, **kwargs):
    # ...
```

For the function in `climada.hazard.tc_tracks_synth`, I think we can rename without alias to `calc_perturbed_trajectories` because it's not really part of the official API.

Reply:
Thanks, done in 6e24d67e8a0503417d59de7bf6460cad6d60bcac.